README.md

```diff
@@ -1,40 +1,87 @@
-# Gulliver
+# Gulliver 
 
-A PWA Directory example.
+[Gulliver](https://pwa-directory.appspot.com/) is a directory of [Progressive Web Apps](https://infrequently.org/2016/09/what-exactly-makes-something-a-progressive-web-app/).
+
+## Contents
+
+In Gulliver's landing page you can browse the set of currently registered PWAs as depicted in the following landing page snapshot:
+
+![Screenshot](img/gulliver-landing-page.png)
+
+If you click on any particular PWA you want, Gulliver takes you to a detail page showing the results of an evaluation done on that specific PWA using the  [Lighthouse PWA Analyzer](https://www.youtube.com/watch?v=KiV2p46rWjU) tool (Details page #1), and a view of the associated [web app manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) file  for the application (Details Page #2):
+
+Details Page #1            |  Details Page #2
+:-------------------------:|:-------------------------:
+![](img/gulliver-details-one.png)  |  ![](img/gulliver-details-two.png)
+
+Gulliver itself has been implemented as a PWA; therefore it is designed to work well in any kind of device, including desktop web browsers (see landing page), and in mobile devices (see details page).
 
 ## Requirements
 
+Gulliver's is built using the [ExpressJS](https://expressjs.com/) web framework for Node.js, and uses the [Google Cloud Platform](https://cloud.google.com/) (GCP) for computing and storage services. Therefore,the following components are required to run the project:
+
 1. [NodeJS](https://nodejs.org). 
-1. [Google Cloud SDK](https://cloud.google.com/sdk/)
-1. A [Google Cloud Console](https://console.cloud.google.com/) project.
-1. Memcached installed and running on localhost:11211. [Installation instructions](https://cloud.google.com/appengine/docs/flexible/nodejs/caching-application-data).
 
-## Installation
+1. [Google Cloud SDK](https://cloud.google.com/sdk/). A set of tools for GCP that you can use to access the Google Compute Engine and the Google Cloud Storage, which are two components of GCP used by Gulliver.
+
+1. A [Google Cloud Console](https://console.cloud.google.com/) project. A GCC project forms the basis of accessing the GCP. 
+
+1. [Memcached](https://memcached.org/) : a distributed memory object caching system.
```

Comment:
We're using a period for Node JS, Google Cloud SDK and the console, and a colon for memcached. Maybe we could standardize on either the period or the colon